ICRCs/ICRC-97/ICRC-97.md

```diff
@@ -0,0 +1,95 @@
+| ICRC |                   Title                    |                                      Author                                      |                Discussions                | Status |      Type       | Category |  Created   |
+|:----:|:------------------------------------------:|:--------------------------------------------------------------------------------:|:-----------------------------------------:|:------:|:---------------:|:--------:|:----------:|
+|  97  | Non-Fungible Token (NFT) Metadata Standard | Thomas (@sea-snake), Austin Fatheree (@skilesare), Dieter Sommer (@dietersommer) | https://github.com/dfinity/ICRC/issues/97 | Draft  | Standards Track |          | 2024-08-13 |
+
+# ICRC-97: Non-Fungible Token (NFT) Metadata Standard
+
+ICRC-97 is a metadata standard for the implementation of Non-Fungible Tokens (NFTs) on
+the [Internet Computer](https://internetcomputer.org).
+
+The standard is designed to allow for multiple assets with different data types with various purposes. Where additional
+asset purposes and attribute display types can be standardized in ICRC-97 extensions.
+
```

Comment:
Should we add the main uses of the standard here as motivation, e.g., that it allows marketplaces to properly show the NFTs?

Reply:
Good point, added!